e2e/nomostest/testpredicates/predicates.go

```diff
@@ -1171,3 +1171,36 @@ func RootSyncSpecOverrideEquals(expected *v1beta1.RootSyncOverrideSpec) Predicat
 		return nil
 	}
 }
+
+func subjectsEqual(want []string, got []rbacv1.Subject) error {
+	if len(got) != len(want) {
+		return errors.Errorf("want %v subjects; got %v", want, got)
+	}
+
+	found := make(map[string]bool)
+	for _, subj := range got {
+		found[subj.Name] = true
+	}
+	for _, name := range want {
+		if !found[name] {
+			return errors.Errorf("missing subject %q", name)
+		}
+	}
+
+	return nil
+}
+
+// ClusterRoleBindingHasSubjects checks that the ClusterRoleBinding has a list
+// of subjects whose names match the specified list of names.
+func ClusterRoleBindingHasSubjects(subjects ...string) func(o client.Object) error {
```

Comment:
Might make sense to name this SubjectsEqual, instead of HasSubjects, since it's validating an exact match, not a subset.

Reply:
Done